apis/python/tests/test_dataframe_index_columns.py

```diff
@@ -54,18 +55,27 @@ def arrow_table():
 
 
 @pytest.mark.parametrize(
-    "name,index_column_names,coords,expecteds",
+    "name,index_column_names,domains,coords,expecteds",
```

Comment:
I believe this is missing a test for the case where the domain of a `soma_joinid` indexed dataframe has a user-specified domain that is illegal.

I.e., something like `index_column_names=['soma_joinid']` and `domains=[(-1000,-1)]`

Reply:
committed